### PR TITLE
Progressive bar

### PR DESCRIPTION
**Pull request by veeranjaneyuludavuluri**
Pulled login modifications branch into progressive-bar
Removed scroll to the body to eliminate external scroll of the body

### CODE REVIEW

**Comment by Sina-pz** [comment]
try to use the style from bass.scss

**Review by Sina-pz — APPROVED** [review]

File: src/app/reusable/progressive-bar/progressive-bar.component.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+})
+export class ProgressiveBarComponent implements OnInit {
+
+  @Input() color: string = '';
+  @Input() size: string = '';
+  @Input() secondarySize: string = '';
+  public totalSize: number = 0;
+
+  public progressiveStatus: any = [
+    { sucess: '#198515' },
```
[the remark this answers is not in the document]
Comment (veeranjaneyuludavuluri) [reply src/app/reusable/progressive-bar/progressive-bar.component.ts:16]: readProperty(name: string): string {
        let bodyStyles = window.getComputedStyle(document.body);
        return bodyStyles.getPropertyValue(PREFIX + name);
    }
    
    As per my knowledge we need to get scss values into ts by this process but I don't think that accessing the document body and window objects directly will approve the pen testing policies 
Please let me know how to access the values

[the remark this answers is not in the document]
Comment (dhaval2989) [reply src/app/reusable/progressive-bar/progressive-bar.component.ts:16]: ok, we will create a separate task.

**Review by dhaval2989 — APPROVED** [review]
good job.

**Pull request merged**